BlueprintUI/Tests/ConstrainedPercentTests.swift

```diff
@@ -0,0 +1,44 @@
+import XCTest
+import BlueprintUI
+
+class ConstrainedPercentTests: XCTestCase {
```

Comment:
It would be good to add a test using a Flexible Element like we do in constrained size, to ensure that elements that "re-flow" (like text, where the width affects the height as the text re-flows) are handled in a reasonable way: https://github.com/square/Blueprint/blob/main/BlueprintUI/Tests/ConstrainedSizeTests.swift#L340

Reply:
So, I spent some time looking at this example, and it doesn't seem to apply. `ConstrainedSizeFraction` currently functions like `ConstainedSize's` `.absolute` case in that it overrides element measurements when a constraint exists in a given dimension.

Please correct me if I'm off base here!

Reply:
Got it, I think that makes sense